tail_test.go

```diff
@@ -95,308 +95,308 @@ func TestStop(t *testing.T) {
 	tail.Cleanup()
 }
 
-func TestStopAtEOF(_t *testing.T) {
-	t := NewTailTest("maxlinesize", _t)
-	t.CreateFile("test.txt", "hello\nthere\nworld\n")
-	tail := t.StartTail("test.txt", Config{Follow: true, Location: nil})
+func TestStopAtEOF(t *testing.T) {
+	tailTest := NewTailTest("maxlinesize", t)
+	tailTest.CreateFile("test.txt", "hello\nthere\nworld\n")
+	tail := tailTest.StartTail("test.txt", Config{Follow: true, Location: nil})
 
 	// read "hello"
 	<-tail.Lines
 
 	<-time.After(100 * time.Millisecond)
```

Comment:
What is this wait for? You just read out a line, so it's definitely ready, right?